guac/certifier-scorecard.md

```diff
@@ -0,0 +1,233 @@
+| `--scorecard-domain-prefix` | Domain prefix for repository URLs (API mode only) | `github.com`                         |
+| `--scorecard-http-timeout`  | HTTP timeout for API requests (API mode only)     | `30s`                                |
+
+### Global Flags
+
+| Flag                   | Description                                                                      | Default                                 |
+| ---------------------- | -------------------------------------------------------------------------------- | --------------------------------------- |
+| `--gql-addr string`    | Endpoint used to connect to GraphQL server                                       | `http://localhost:8080/query`           |
+| `--pubsub-addr string` | Address to connect to NATS pubsub service                                        | `nats://localhost:4222`                 |
+| `--blob-addr string`   | Address for blob storage                                                         | `file:///tmp/blobstore?no_tmp_dir=true` |
+| `--header-file string` | A text file containing HTTP headers to send to the GQL server, in RFC 822 format | Not set                                 |
+| `--publish-to-queue`   | Enable publishing to the message queue                                           | true                                    |
+
+## Usage Examples
+
+### API-Based Scorecard Fetcher (Default)
+
+```bash
+# Use API-based fetcher with default settings
+guaccollect scorecard \
+  --gql-addr=http://localhost:8080/query \
+  --pubsub-addr=nats://localhost:4222 \
+```
+
+```bash
+# Use API-based fetcher with custom settings
+guaccollect scorecard \
+  --scorecard-fetcher-type=api \
+  --scorecard-api-base=https://api.securityscorecards.dev \
+  --scorecard-domain-prefix=github.com \
+  --scorecard-http-timeout=60s \
+  --gql-addr=http://localhost:8080/query \
+  --pubsub-addr=nats://localhost:4222 \
+```
+
+### Local Scorecard Fetcher
+
+```bash
+# Set GitHub token (required for local mode)
+export GITHUB_AUTH_TOKEN=your_github_token
+```
+
+```bash
+# Use local scorecard library
+guaccollect scorecard \
+  --scorecard-fetcher-type=local \
+  --gql-addr=http://localhost:8080/query \
+  --pubsub-addr=nats://localhost:4222 \
+```
+
+### Polling Mode
+
+```bash
+# Enable polling with custom interval
+guaccollect scorecard \
+  --service-poll \
+  --interval=10m \
+  --scorecard-fetcher-type=api
+```
+
+## Prerequisites
+
+### GitHub Token Setup (Local Mode Only)
+
+1. Create a GitHub Personal Access Token:
+   - Go to GitHub Settings → Developer settings → Personal access tokens
+   - Generate new token with `repo` and `read:org` scopes
+
+2. Set environment variable:
+   ```bash
+   export GITHUB_AUTH_TOKEN=ghp_your_token_here
+   ```
+
+## Limitations
+
+### API Mode Limitations
+
+- Limited to repositories already analyzed by OpenSSF Scorecard
+- No control over check configuration
+
+### Local Mode Limitations
```

Comment:
It might be good to explicitly mention that local mode does not work for projects hosted on other platforms. This is implicit from the "Requires GitHub authentication toke", but it'd better to be explicit about that constraint.